aws/table_aws_cloudwatch_log_group.go

```diff
@@ -200,12 +231,16 @@ func getLogGroupTagging(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 		return nil, err
 	}
 
-	params := &cloudwatchlogs.ListTagsLogGroupInput{
-		LogGroupName: logGroup.LogGroupName,
+	// Removing extra characters from the end of Arn
+	logGroupArn := string(*logGroup.Arn)
+	logGroupArn = logGroupArn[0 : len(logGroupArn)-2]
```

Comment:
https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/iam-access-control-overview-cwl.html
@ParthaI We need to pass the split ARN in this case since the output format of the log-group ARN is in the format mentioned in the document.

```
Both of the following are used. The second one, with the * at the end, is what is returned by the describe-log-groups CLI command and the DescribeLogGroups API.

arn:aws:logs:region:account-id:log-group:log_group_name

arn:aws:logs:region:account-id:log-group:log_group_name:*
```

Reply:
What transforms do we use for similar columns in other tables, if any? Also when querying the column, were you able to interact with the column as if it were JSONB, e.g., use Postgres JSONB functions, access specific elements with ->?